### DISCUSSION TITLE
TuringCourses/math_phys/physics_experiment2/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#105] [General]
# TuringCourses/math_phys/physics_experiment2/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/math_phys/physics_experiment2/

**Comment by MTDickens** [comment]
混合实验、小课题、科创项目是否有推荐呢？
然后为什么说“最坏的是 14 个小实验”呢？

**Reply by TonyCrane** [reply]
十四个小实验最麻烦，每周都要花时间做。为期越长的可以摆的时间多，更轻松点

没什么好推荐的，选课网站从下往上，能选上就是赢（

**Comment by Flyburg** [comment]
本人选择的是陈水桥老师指导的科创项目《纹影成像研究》。这个实验没有给定方向，而我们花了将近 1 个月的时间来想选题（老师对于选题的创新性要求是比较高的，很多最开始的想法都被毙掉了），十月中旬才着手做实验，所以浪费了较多的时间，希望大家引以为戒（

但是陈老师给分是非常非常nice的~

**Comment by Fairicle** [comment]
仅从 22 级的上课情况来看，无论是哪个老师，大实验的给分普遍而言还是很好的。即使没有做出什么成果，只要付出了自己的努力，老师基本上都能给到 90+ 的分数。如果能够比较好的完成实验或者做出一定成果更不必担心分数。

陈水桥老师比较刀子嘴豆腐心，每节课基本都会来了解进度并给你挑刺找各种毛病，但是给分非常非常好。

**Reply by lhxcs** [reply]
郑远老师人非常好，在实验过程中遇到任何问题都可以和他讨论，完全没有老师的架子，他会非常耐心地指导实验的进行，最后的大论文还会给评语，虽然直切要害但是非常温柔，最后给分也很好。

**Comment by 626lt** [comment]
我选的肖婷老师的7周小课题《磁场测量的应用》，老师人很好，会积极地沟通和帮助指导，也比较温和，不会很尖锐地批评你。其给分挺好，但不会给超过95的分数，一个小组的分数基本上都是一样的。而且给分来看也是根据你们小组做了多少工作来说的。我们小组由于各种原因最后只有三个人来完成五个人的任务，最后做的工作量其实也只要一个人就可以完成了，但是最后也是得到了94分的分数。据了解在冬学期同样的小课题只是在我们工作的基础上做了简单的拓展也得到了90的分数，所以有些小课题整体的工作量也不是很大。我个人比较推荐选小课题或者14周的科创项目，这种实验更加深入而且没有每周一次的实验报告压力，对此相信经历过普物实验一的各位都很清楚孰优孰劣。
